### PR TITLE
Added the code of BubbleSort Algorithm in C++

### PR DESCRIPTION
**Pull request by 2G-Afroz**
Added the code for BubbleSort algorithm in C++.

### CODE REVIEW

**Comment by Alaz-Oz** [comment]
There shouldn't be a.out file. bcoz, That is platform dependent, and won't work on other platforms. & also the repo will only contain code, not the binaries.
Tasks to be done before merging:
- Remove `a.out` file
- Change `bubbleSort.cpp` to `BubbleSort.cpp`

+ We could have multiple examples demonstrating the algorithm. Yours uses vector, mine uses static array.

We need a way to organise multiple example code.
Let's say we name all the examples like this `BubbleSort1.cpp`, `BubbleSort2.cpp` and so on,
then we could store multiple examples at once.
## What do you say?

File: Algorithms/Sort/BubbleSort/a.out
Comment (Alaz-Oz) [inline Algorithms/Sort/BubbleSort/a.out:1]: Remove this binary, and we are good to go

**Comment by Alaz-Oz** [comment]
It would be a good idea to create another pull request that is fresh, so that there is no residual a.out. (Which will be downloaded when we `git pull` or `git clone` our repo. There will be extra overhead `a.out` there.

So create a new branch --> Commit only the `BubbleSort2.cpp` --> Create PR

**Comment by Alaz-Oz** [comment]
Also,
## Use tabs as your previous commits.

Don't use space for the indentations use tabs instead. (As your previous commits have tabs, and I adopted using tabs in the source codes (NOT in the pseudocode).

### Tabs in source code & 4 spaces in pseudocode
(If you want to change this format, create a discussion & justify your answer)
Although I would agree using 4 spaces everywhere in the code and pseudocode

**Comment by 2G-Afroz** [comment]
> There shouldn't be a.out file. bcoz, That is platform dependent, and won't work on other platforms. & also the repo will only contain code, not the binaries. Tasks to be done before merging:
> 
> * Remove `a.out` file
> * Change `bubbleSort.cpp` to `BubbleSort.cpp`
> 
> * We could have multiple examples demonstrating the algorithm. Yours uses vector, mine uses static array.
> 
> We need a way to organise multiple example code. Let's say we name all the examples like this `BubbleSort1.cpp`, `BubbleSort2.cpp` and so on, then we could store multiple examples at once.
> 
> ## What do you say?

The code is just for example, Which explains the algorithm in a specific Programming Language.
We are not teaching how to use different datatypes of a language. If the user can't use different datatypes in an algorithm then they should learn that first.

**Comment by Alaz-Oz** [comment]
We should move to writing the generic version of the algorithms then, covering the most datatypes possible for that algorithms.

**Pull request closed without merging**